src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1059,6 +1059,15 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
             $container->setDefinition($name, $definition);
         }
 
+        // For the PropertyAccessor, caching on the filesystem is slower than disabling the PSR-6 cache
+        if ($config['system'] !== 'cache.adapter.filesystem') {
```

Comment:
This strategy won't work if #16838 is merged (I hope it will be merged :) ). Instead, we'd either need a factory, or maybe not configure this pool at all by default. Or maybe just to blacklist the FilesystemAdapter in the constructor (i.e. set it to `null` when this type is provided)?